metrics/cli/test/gcloud/adapter/gcloud_cli_service_adapter_test.dart

```diff
@@ -59,143 +60,213 @@ void main() {
-        ]);
+        verify(gcloudCli.createProjectApp(region, projectId)).called(once);
       },
     );
 
     test(
-      ".createProject() requests the region from the user",
+      ".createProject() enables Firestore API for the project with the generated project id",
       () async {
-        await gcloudAdapter.createProject();
+        final projectId = await gcloudService.createProject();
 
-        verify(prompter.prompt(GcloudStrings.enterRegionName)).called(once);
+        verify(gcloudCli.enableFirestoreApi(projectId)).called(once);
       },
     );
 
     test(
-      ".createProject() requests the region from the user before creating the project application",
+      ".createProject() creates database with the generated id and the given region",
       () async {
         whenEnterRegionPrompt().thenReturn(region);
 
-        final projectId = await gcloudAdapter.createProject();
+        final projectId = await gcloudService.createProject();
 
-        verifyInOrder([
-          prompter.prompt(GcloudStrings.enterRegionName),
-          gcloudCli.createProjectApp(region, projectId),
-        ]);
+        verify(gcloudCli.createDatabase(region, projectId)).called(once);
       },
     );
 
     test(
-      ".createProject() creates the project app with the given region and the generated project id",
+      ".createProject() returns the identifier of the created project",
       () async {
-        whenEnterRegionPrompt().thenReturn(region);
+        final projectId = await gcloudService.createProject();
 
-        final projectId = await gcloudAdapter.createProject();
+        expect(projectId, isNotNull);
+        verify(gcloudCli.createProject(projectId)).called(once);
+      },
+    );
 
-        verify(gcloudCli.createProjectApp(region, projectId)).called(once);
+    test(
+      ".version() shows the version information",
+      () async {
+        await gcloudService.version();
+
+        verify(gcloudCli.version()).called(once);
       },
     );
 
     test(
-      ".createProject() creates the project app before enabling Firestore API",
+      ".login() throws if GCloud CLI throws during the logging",
       () async {
-        whenEnterRegionPrompt().thenReturn(region);
+        when(gcloudCli.login()).thenAnswer((_) => Future.error(stateError));
 
-        final projectId = await gcloudAdapter.createProject();
+        await expectLater(gcloudService.login(), throwsStateError);
 
-        verifyInOrder([
-          gcloudCli.createProjectApp(region, projectId),
-          gcloudCli.enableFirestoreApi(projectId),
-        ]);
+        verify(gcloudCli.login()).called(once);
+
+        verifyNoMoreInteractions(gcloudCli);
+        verifyNoMoreInteractions(prompter);
       },
     );
 
     test(
-      ".createProject() enables Firestore API for the project with the generated project id",
+      ".createProject() doesn't call any other methods if GCloud CLI throws during the project creation",
```

Comment:
```suggestion
      ".createProject() stops the project creation process if GCloud CLI throws during the project creation",
```
What do you think? Please, apply this suggestion to all similar tests.